es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log('LearningFuze: ' + address);
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log('Bio: ' + bio);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit}${item}`;
+
+    console.log('I have a ' + item);
+    console.log('I have an ' + fruit);
```

Comment:
you should be using `,` here instead of `+`

Reply:
<img width="268" alt="image" src="https://user-images.githubusercontent.com/113648549/204364221-eb2d54d1-777c-41d2-8bdc-2b32e0acb6f2.png">